src/CodeStyle/verify-twig-template-usage.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+#
+# finds all (potential) usages of twig templates and checks if they exist
+
+set -o nounset -o errexit
+
+atExitReport () {
+    rc=$?
+    if [ 0 -ne $rc ]
+    then
+        printf '  ** failed with %s' "$rc"
+        [ -n "$1" ] && printf ' on line %s' "$1"
+        printf '\n'
+    fi >&2
+}
+
+trap 'atExitReport "${LINENO:-}"' EXIT
+
+noArgs=
+if [ 0 -eq $# ]
+then
+    set -- src/ templates/
+    noArgs=1
+fi
+
+colorMatch() {
+    grep --color=auto -e '' -e "[^:/ ]$1."
+}
+
+matchD='"[^"][^"]*\.twig"'
+matchS="'[^'][^']*\\.twig'"
+
+{
+    grep -r --line-number '--include=*.php' '--include=*.twig' -e "$matchD" -e "$matchS" "$@"
+    if [ -n "$noArgs" ]
+    then
```

Comment:
add comment like: when checking all (=noArgs), then also check for templates in config files

Reply:
done

Reply:
done